test/test_jit.py

```diff
@@ -10022,14 +10022,22 @@ def func():
         def isBool(arg):
             return type(arg) == bool or (type(arg) == str and "torch.bool" in arg)
 
+        def isIntegral(arg):
+            return type(arg) == int or (type(arg) == str and "torch.int" in arg)
+
         for op in ops:
             for first_arg in args:
                 for second_arg in args:
                     # subtract not supported for bool
                     if (op == 'sub' or op == 'div') and (isBool(first_arg) or isBool(second_arg)):
                         continue
-                    # div not implemneted correctly for mixed-type or in params
-                    if (op == 'div' and (type(first_arg) != type(second_arg) or type(first_arg) == int)):
+                    # Note: div is not implemented correctly for mixed-type
+                    # or int params
+                    # TODO: restore integral division once it's re-enabled
+                    if (op == 'div' and (type(first_arg) != type(second_arg)
```

Comment:
see below, it might be nicer to just run true_divide through this so you don't need to change the logic.

Reply:
Sounds good. Moved this out of this PR so I'll follow-up with it.